HELP.md

```diff
@@ -0,0 +1,19 @@
+# Getting Started
```

Comment:
We don't need this file in Git. You can keep it on your setup, to find the resources you might need later.

Reply:
Done, thanks!